roles/reproducer/tasks/validations.yml

```diff
@@ -36,7 +36,7 @@
       - ansible_distribution in cifmw_reproducer_supported_hypervisor_os
       - ansible_distribution_version | float  >= cifmw_reproducer_supported_hypervisor_os[ansible_distribution].minimum_version
     msg: >-
-      Please ensure host: {{ cifmw_target_host }} is running a supported OS.
+      Please ensure host: {{ cifmw_target_host | default(ansible_host) }} is running a supported OS.
```

Comment:
I'd rather use `| default('localhost')`
reason:
- ansible_host may not be set (it comes from the inventory)
- at that point, the only reason to not have `cifmw_target_host` is a run against `localhost`
- default value for `cifmw_target_host` is `localhost` in all playbooks

Reply:
Ack, I wasnt sure about ansible_host being safe(r), thanks fixed.